### ISSUE TITLE
Could not install version 6.1.3

### ISSUE THREAD

**Issue by subhashisrouth** [AppsFlyerSDK/appsflyer-react-native-plugin#231]
# Report

## Plugin Version
`6.1.3`

## On what Platform are you having the issue?

ℹ Please replace these two lines with the SDK version.
`iOS AppsFlyerSDK v6.1.3`

## What did you do?

ℹ Please replace these two lines with what you did.
`npm install react-native-appsflyer --save`
`cd ios && pod install`  

## What did you expect to happen?

ℹ To Successfully install the plugin 

## What happened instead?

`CocoaPods could not find compatible versions for pod "AppsFlyerFramework":`
  `In Podfile:`
    `react-native-appsflyer (from '../node_modules/react-native-appsflyer') was resolved to 6.1.30, which depends on`
      `AppsFlyerFramework (= 6.1.3)`

## Please provide any other relevant information.

ℹ 
I installed the version 6.0.50 Successfully
But want to upgrade the version to 6.3.1
But giving error, as  AppsFlyerFramework version 6.1.3 could not be found from Cocapods.

![Screenshot 2021-01-07 at 6 40 31 PM](https://user-images.githubusercontent.com/6926322/103896331-d4219380-5117-11eb-9451-07f14989db1d.png)

**Comment by amit-kremer93** [comment]
Hi @subhashisrouth. try to remove the plugin completely with:
`npm uninstall react-native-appsflyer`
go to `ios` folder and delete `Podfile.lock`
run `pod update`
then back to the root folder and run `npm install react-native-appsflyer` and in `ios` folder `pod install`

**Comment by DevSubhashis** [comment]
Thanks @amit-kremer93 for replying, I have tried but did not work.

**Comment by DevSubhashis** [comment]
I also tried on a fresh new React Native project, its giving me same error. I think somehow cocapods can not find/download AppsFlyerFramework version 6.1.3

**Comment by rborn** [comment]
@DevSubhashis did you try `pod repo update` ?

**Comment by vinhtnk** [comment]
try: `pod update AppsFlyerFramework`

**Comment by amit-kremer93** [comment]
@subhashisrouth did you manage to install the plugin?

**Comment by shinriyo** [comment]
```
firebase_storage: Using Firebase SDK version '6.33.0' defined in 'firebase_core'
[!] CocoaPods could not find compatible versions for pod "AppsFlyerFramework":
  In snapshot (Podfile.lock):
    AppsFlyerFramework (= 6.2.1)

  In Podfile:
    appsflyer_sdk (from `.symlinks/plugins/appsflyer_sdk/ios`) was resolved to 6.2.0, which depends on
      AppsFlyerFramework (= 6.2.3)


You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * changed the constraints of dependency `AppsFlyerFramework` inside your development pod `appsflyer_sdk`.
   You should run `pod update AppsFlyerFramework` to apply changes you've made.

```

**Comment by skibaalex** [comment]
same issue.
@subhashisrouth did you manage to solve the problem?
